/controller_frequency: 10

CollvoidLocalPlanner:
  simulation_mode: true
  max_neighbors: 10
  neighbor_dist: 3.0
  time_horizon: 10.0
  time_horizon_obst: 10.0
  threshold_last_seen: 1.0

  yaw_goal_tolerance: 0.05
  xy_goal_tolerance: 0.10
#  latch_xy_goal_tolerance: true
  global_frame: /map
  wheel_base: 0.25
  time_to_holo: 0.4
  min_error_holo: 0.01
  max_error_holo: 0.15
  holo_robot: false
  max_vel_x: 0.5
  max_vel_th: 1.5
  min_vel_x: 0.1
  min_vel_th: 0.1
  min_vel_y: 0.01
  max_vel_y: 0.1
  acc_lim_x: 1.0
  acc_lim_y: 0.0
  acc_lim_th: 1.0
  radius: 0.17
  inscribed_radius: 0.17

TrajectoryPlannerROS:
  max_vel_x: 0.50
  min_vel_x: 0.10
  max_rotational_vel: 1.5
  min_in_place_rotational_vel: 0.1
  acc_lim_th: 0.75
  acc_lim_x: 0.50
  acc_lim_y: 0.50
  holonomic_robot: false
  yaw_goal_tolerance: 0.05
  xy_goal_tolerance: 0.1
  goal_distance_bias: 0.8
  path_distance_bias: 0.6
  sim_time: 1.5
  heading_lookahead: 0.325
  oscillation_reset_dist: 0.05
  vx_samples: 6
  vtheta_samples: 20
  dwa: false
 
local_costmap:
  global_frame: /odom
  robot_base_frame: /base_link
  update_frequency: 3.0
  publish_frequency: 2.0
  static_map: false
  rolling_window: true
  width: 4.0
  height: 4.0
  resolution: 0.05
  map_type: voxel
  z_voxels: 10
  unknown_threshold: 8
  obstacle_range: 2.5
  raytrace_range: 3.0
  robot_radius: 0.17
#  footprint: []#[[0.17, 0.17], [-0.17, 0.17], [-0.17, -0.17], [0.17, -0.17]]
  inflation_radius: 0.18
  observation_sources: laser_scan_sensor
  laser_scan_sensor: {sensor_frame: /base_laser_link, data_type: LaserScan, topic: /base_scan, marking: true, clearing: true}
  #point_cloud_sensor: {sensor_frame: frame_name, data_type: PointCloud, topic: topic_name, marking: true, clearing: true}

global_costmap:
  global_frame: /map
  robot_base_frame: /base_link
  update_frequency: 3.0
  publish_frequency: 2.0
  static_map: true
  map_type: voxel
  z_voxels: 10
  unknown_threshold: 8
  transform_tolerance: 0.5
 
  obstacle_range: 2.5
  raytrace_range: 3.0
  robot_radius: 0.17
 # footprint: []#[[0.17, 0.17], [-0.17, 0.17], [-0.17, -0.17], [0.17, -0.17]]
  inflation_radius: 0.20
  observation_sources: laser_scan_sensor
  laser_scan_sensor: {sensor_frame: /base_laser_link, data_type: LaserScan, topic: /base_scan, marking: true, clearing: true}
